### ISSUE TITLE
Add hyperclick

### ISSUE THREAD

**Issue by NgKhanh** [DFreds/code-peek-atom#25]
I thinks it'll be awesome if combined with hyperclick
https://github.com/facebooknuclide/hyperclick

**Comment by DFreds** [comment]
I agree although I'm not quite sure how it would work or behave. If you have any suggestions or want to try it out yourself, feel free!

**Comment by DFreds** [comment]
Someone can feel free to try this, but I'm going to label it as wontfix.
